data_generators/generators.py

```diff
@@ -82,8 +82,8 @@ class RandomMeeting(Printable):
     def __init__(self, mentee_id, mentor_id):
         self.meeting_id = generate_uuid(16)
         self.meeting_topic = choice(topics)
-        self.meeting_start_time = datetime.now().isoformat()
-        self.meeting_end_time = datetime.now().isoformat()
+        self.meeting_start_time = random_datetime(datetime(2022, 8, 1), datetime(2022, 8, 2))
+        self.meeting_end_time = self.meeting_start_time + timedelta(hours=1)
```

Comment:
Do we want all the random meetings to happen on the same two days?

Reply:
I have resent the push to be the same datetime length as earlier, not just over one day.